English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_less_83_88.txt

```diff
@@ -0,0 +1 @@
+Which of the the number 83 and 88 is greater than 85? Drag and drop the correct number to make number sentence true.
```

Comment:
Which **among** the ~~the~~ numbers 83 and 88 is greater than 85?

Reply:
Updated the transcript.